src/Query/Filter/TextField.ts

```diff
@@ -21,13 +21,12 @@ export abstract class TextField extends Field {
 
         // Construct an IStringMatcher for this filter, or return
         // if the inputs are invalid.
-        const filterMethod = match[1];
-        const searchString = match[2];
+        const [_, operator, value] = match;
```

Comment:
I wasn't able to guess the purpose of `value` from the name. The previous variable was `searchString` seemed a little more informative.

Perhaps add a comment saying what `value` is?

Reply:
(or perhaps add a more self-explanatory name?)

Reply:
Later... I wonder whether we have different mental models of what's going in...
I'm more interested in understanding how you are thinking of what's going in, than the changing of variable names!

Reply:
Actually the original reason to use shorter names was that I wanted to used named groups in the regex that would correspond to variable names, and didn't want to make the regex look to complicated with these long names. But I found that named groups made everything look more complicated, so I reverted that, but forgot to change the names back.

Other reasons: I prefer the term "operator" to "method" since in the syntax it is used like an operator in the middle between the two operands (the name of the field on the left side and the value of the filter on the right side), not like a method. Also, I don't like the "search" in "searchString" because we're doing filtering here, not searching. And I don't like "String" because it can also be a regex pattern, not a plain string. Maybe I should rename it to "filterValue"?

Reply:
Oh that was all really useful information, thank you very much.

OK, so we pretty much are just used to different vocabulary for all this, but it's not worth agonising over.

`filterValue` is an improvement - thanks, that would be fine!

Reply:
Ok, I like that better, too. Also named it `filterOperator` instead of just `operator`. It's more verbose now, but clearer.

Side note: Should we rename `filterRegexp` to `filterRegExp` because `RegExp` is the spelling in JavaScript?

Reply:
> Ok, I like that better, too. Also named it `filterOperator` instead of just `operator`. It's more verbose now, but clearer.

I like that. Thank you.

> Side note: Should we rename `filterRegexp` to `filterRegExp` because `RegExp` is the spelling in JavaScript?

I have no preference.

It's not something I would notice, as I don't know JS that well, and the information content is the same either way, which is what I most care about.

so - it's up to you! 😄

Reply:
Ok, made that change (VC Code makes such renaming very easy). I think `filterRegExp(): RegExp` looks much better than `filterRegexp(): RegExp`.

Reply:
> Ok, made that change (VC Code makes such renaming very easy). I think `filterRegExp(): RegExp` looks much better than `filterRegexp(): RegExp`.

Oh, I thought you were renaming variables. Never mind. I'll edit the title of the PR to add the additional change.